Gemfile.lock

```diff
@@ -16,8 +16,12 @@ GIT
       i18n (>= 1.6, < 2)
 
 GEM
-  remote: https://rubygems.org/
   remote: https://rails-assets.org/
+  specs:
+    rails-assets-bulma (0.7.4)
```

Comment:
Is it used anywhere? If not, you could remove it from [Gemfile](https://github.com/cybergizer-hq/retrospective/blob/master/Gemfile#L87)

Reply:
https://github.com/cybergizer-hq/retrospective/blob/fa53f3ff06cebbdd317d12f8a46e590b8d08f42f/Gemfile#L87

it seems someone has merged this, but didn't commit it to the Gemfile.lock, that's why we get it here.